src/utils/email.py

```diff
@@ -15,3 +17,33 @@ def codalab_send_mail(context_data, to_email, html_file, text_file, subject, fro
     message = EmailMultiAlternatives(subject, text, from_email=from_email, to=to_email)
     message.attach_alternative(html, 'text/html')
     message.send()
+
+
+def sanitize(content):
+    tags = [
+        'img',
+        'p',
+        'br',
+        'h1',
+        'h2',
+        'h3',
+        'h4',
+        'h5',
+        'h6',
+    ]
+    attrs = {'img': ['src']}
+    return bleach.clean(
+        content,
+        tags=tags,
+        attributes=attrs
+    )
+
+
+def codalab_send_markdown_email(subject, markdown_content, recipient_list, from_email=None):
+    from_email = from_email if from_email else settings.DEFAULT_FROM_EMAIL
+    message = sanitize(markdown_content)
```

Comment:
In terms of what the rendered markdown will look like? almost certainly... since bleach is a white-list based sanitizer with a _meager_ standard whitelist, I have almost definitely not whitelisted additional tags that we should be... kinda hard (for me) to anticipate all the tags and attributes that we should whitelist.

Reply:
In terms of what the _text_ version of the markdown will look like -- I think this is so unused we shouldn't worry about it for now